Sources/SpeziAccessGuard/CodeViews/SetCodeView.swift

```diff
@@ -120,10 +149,11 @@ struct SetCodeView: View {
                         }
                     )
                 }
-                    .frame(height: 80)
+                .frame(height: 80)
             }
-                .transition(.opacity)
-                .navigationBarBackButtonHidden()
+            .transition(.opacity)
+            .navigationBarBackButtonHidden()
+            
```

Comment:
Please avoid any extensive diffs that do not change any implementation elements. We generally indent modifiers by one level and don't have empty lines in switch statements. Same formatting feedback applies to a few other places in this PR.
```suggestion
                .transition(.opacity)
                .navigationBarBackButtonHidden()
```